stubs/tensorflow/tensorflow/_aliases.pyi

```diff
@@ -1,30 +1,55 @@
 # Commonly used type aliases.
-# Everything in this module is private for stubs. There is no runtime
-# equivalent.
+# Everything in this module is private for stubs. There is no runtime equivalent.
 
-from collections.abc import Mapping, Sequence
+from collections.abc import Iterable, Mapping, Sequence
 from typing import Any, Protocol, TypeVar
 from typing_extensions import TypeAlias
 
-import numpy
+import numpy as np
+import numpy.typing as npt
 import tensorflow as tf
+from tensorflow.keras.layers import InputSpec
 
-_T1 = TypeVar("_T1")
-ContainerGeneric: TypeAlias = Mapping[str, ContainerGeneric[_T1]] | Sequence[ContainerGeneric[_T1]] | _T1
+_T = TypeVar("_T")
+_ContainerGeneric: TypeAlias = Mapping[str, _ContainerGeneric[_T]] | Sequence[_ContainerGeneric[_T]] | _T
 
-TensorLike: TypeAlias = tf.Tensor | tf.RaggedTensor | tf.SparseTensor
-Gradients: TypeAlias = tf.Tensor | tf.IndexedSlices
-
-ContainerTensorsLike: TypeAlias = ContainerGeneric[TensorLike]
-ContainerTensors: TypeAlias = ContainerGeneric[tf.Tensor]
-ContainerGradients: TypeAlias = ContainerGeneric[Gradients]
-
-AnyArray: TypeAlias = numpy.ndarray[Any, Any]
+_TensorLike: TypeAlias = tf.Tensor | tf.RaggedTensor | tf.SparseTensor
+# _SparseTensorLike: TypeAlias = tf.Tensor | tf.SparseTensor
+# _RaggedTensorLike: TypeAlias = tf.Tensor | tf.RaggedTensor
+# _RaggedTensorLikeT = TypeVar("_RaggedTensorLikeT", tf.Tensor, tf.RaggedTensor)
+_Gradients: TypeAlias = tf.Tensor | tf.IndexedSlices
 
 class _KerasSerializable1(Protocol):
     def get_config(self) -> dict[str, Any]: ...
 
 class _KerasSerializable2(Protocol):
     __name__: str
 
-KerasSerializable: TypeAlias = _KerasSerializable1 | _KerasSerializable2
+_KerasSerializable: TypeAlias = _KerasSerializable1 | _KerasSerializable2
```

Comment:
I think since you've unified aliases to all live here consistently they can all lose underscore prefix.

Except for that the PR looks good to me.

Reply:
Done in c7d65fd